script/Arcueid/Day 10/10_00_ARC10_3B.txt

```diff
@@ -198,29 +207,34 @@ Wait a second.
 [sha:7c03dca6e2ab94e4c78a9d355c81fc2b7676d960]{
 -- Page 111, Offset 24539.
 -- かわいい？　とかそういう次元の話じゃねえよ。
-// these next few lines are just... what?
-Cute? That's looking at it all wrong.
+Cute? This isn't simply a matter of cuteness.
 }
 [sha:e37f801b703c4fb7892e41e4d1ef8f7ef38fbd65]{
 -- Page 111, Offset 24540.
 -- こんなの理解の外の話だよ。理性の外の話だよ。
-That's a topic beyond comprehension. A topic beyond reason.
+This is something that defies comprehension.
+ That's beyond reason.
```

Comment:
Hm, I feel like `topic` is kinda awkward though, since it's not really the topic _of discussion_ or anything. I also read this as 
> This is something that defies comprehension. [This is something] that's beyond reason.

and so the _subject_ is still `This`, rather than switching to `that`. If it's an ambiguous parse, maybe something like
> What I've just witnessed defies comprehension. It's beyond all reason.

?

Reply:
The latter works for me